content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -710,21 +727,22 @@ public void checkForReport() {
 					continue;
 				}
 				// Check if any placeholder items need to regenerate report after due date
-				if (PLACEHOLDER_ITEM_REVIEW_SCORE.equals(item.getReviewScore())) {	
+				if (PLACEHOLDER_ITEM_REVIEW_SCORE.equals(item.getReviewScore())) {
```

Comment:
remove whitespace

Reply:
Rename generic PLACEHOLDER_ITEM_REVIEW_SCORE to explain the difference of its function